src/lib/charm/openstack/keystone_saml_mellon.py

```diff
@@ -275,6 +271,13 @@ class KeystoneSAMLMellonCharm(charms_openstack.charm.OpenStackCharm):
     # ownership.
     group = 'www-data'
 
+    @property
+    def restart_map(self):
```

Comment:
Is this necessary?  What does it do that the ``charms.openstack`` built in ``restart_map`` structure does not already do for you?

Reply:
Fixed.